server/api.go

```diff
@@ -80,6 +229,11 @@ func (p *Plugin) handleConfig(w http.ResponseWriter, r *http.Request) {
 }
 
 func (p *Plugin) handleExternalAPIjs(w http.ResponseWriter, r *http.Request) {
+	if p.getConfiguration().UseJaaS {
+		p.proxyExternalAPIjsJaaS(w, r)
```

Comment:
We were more include to use a javascript file that is under control to avoid that a security issue in the JaaS service propagates some how in to the mattermost service. @jupenur any opinion here?